manifests/init.pp

```diff
@@ -238,6 +238,12 @@
 #   port => undef
 # @param ssl_management_port
 #   SSL management port.
+# @param ssl_management_cacert
+#   SSL management cacert. If unset set to ssl_cacert for backwards compatibility.
```

Comment:
How does this get defaulted when unset? Doesn’t seem to happen in either the variables or the template?

Reply:
**ssl_management_cacert** is set to **ssl_cacert** in init.pp.  
this keeps backwards compatibility and the new option to define an alternative management_cacert.

Reply:
I don’t see it being set further down here in `init.pp`, tho, just as an Optional param (which can’t have a default)

Reply:
Sorry, didn’t look far enough right. I don’t think it can be optional, though, then, because of quirks around how Puppet works with optional params with defaults.